### ISSUE TITLE
Implement UI for crash analytics opt-in/out

### ISSUE THREAD

**Issue by pacu** [zcash/secant-ios-wallet#236]
This depends on #237

## Is your feature request related to a problem? Please describe.
In order to ensure a high quality experience for users, we need to collect crash data.

## Describe the solution you'd like
Embed a crash analytics tool in the app.  This tool will need to be configurable, so that users can disable it.

We'll have a separate issue to consider the design of what this opt-in or opt-out looks like.

**Comment by pacu** [comment]
implemented on 0.0.1-39
